app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Room List</h1>
+<% @rooms.each do |room| %>
+  <h3><%= "Suite: #{room.suite}" %></h3>
+  <p><%= "Rate: #{room.rate}" %></p>
+  <p><%= "Hotel: #{room.hotel.name}" %></p>
```

Comment:
Thinking back to the Law of Demeter or the "single-dot" principle, we really don't want to chain on method calls wherever we can avoid it. Create a method on the room model called `hotel_name` and call that instead.